### ISSUE TITLE
Releases and website

### ISSUE THREAD

**Issue by pierrejoye** [libgd/libgd#726]
Good evening,

I synced again website with what has been published to libgd.org.

I failed to get travisci working, it seems it limits the amount of the repository as I can't get it to list website.

I think we may just use github action and migrate it. Thoughts?

**Comment by cmb69** [comment]
Yes, Travis is no longer free (FOSS projects get some credits, but when they're gone …). I agree that switching to GH actions is the way forward.

**Comment by pierrejoye** [comment]
I redid it. Fought too long to try to clone a different repo within a gh action.

- The site uses now a single repository, libgd.github.io.
- master is for the pelican content
- gh-pages is the branch used to publish the site

Remaining tasks:
- add CNAME config as part of the repo
- add github actions, while now, make github does push straight to gh-pages, so much less needed for now.

**Comment by pierrejoye** [comment]
Any objection to delete libgd/website? 

It is not used anymore and I was stupid enough to not read the very doc I wrote about using the new one while releasing 2.3.3.

Besides, what do you think about a standard template and a python script to generate the announcement?

**Comment by vapier** [comment]
I don't like deleting things. we can update the description text & readme to explain what's going on, and then mark this repo as read only.
